tests/mip/test_mip_files.py

```diff
@@ -5,6 +5,26 @@
 
 from trailblazer.mip import files
 
+
+FATHER_AT_DROPOUT = 2.673848
```

Comment:
I think these could go into the conftest as a fixture

Reply:
Good point! Done